app/apollo/resolvers/resource.js

```diff
@@ -21,15 +21,16 @@ const GraphqlFields = require('graphql-fields');
 const { buildSearchForResources, convertStrToTextPropsObj } = require('../utils');
 const { ACTIONS, TYPES } = require('../models/const');
 const { EVENTS, GraphqlPubSub, getStreamingTopic } = require('../subscription');
-const { whoIs, validAuth, getAllowedGroups, getGroupConditionsIncludingEmpty, applyClusterInfoOnResources, NotFoundError } = require ('./common');
+const { whoIs, validAuth, getAllowedGroups, getGroupConditionsIncludingEmpty, NotFoundError } = require ('./common');
 const ObjectId = require('mongoose').Types.ObjectId;
+const { applyQueryFieldsToResources } = require('../utils/applyQueryFields');
 
 const conf = require('../../conf.js').conf;
 const S3ClientClass = require('../../s3/s3Client');
 const url = require('url');
 
 // This is service level search function which does not verify user tag permission
-const commonResourcesSearch = async ({ context, org_id, searchFilter, limit=500, queryFields, sort={created: -1} }) => {
+const commonResourcesSearch = async ({ context, searchFilter, limit=500, queryFields, sort={created: -1} }) => { // eslint-disable-line
```

Comment:
the places in the code that are calling `commonResourcesSearch` are still passing in org_id.  Should those be updated to no longer do that?